src/components/FormBuilder.tsx

```diff
@@ -280,7 +287,7 @@ export class FormBuilder extends React.Component<IFormBuilderProps, {}> {
 
         const component = React.createElement(fieldDef.builder, fieldBuilderProps);
         return (
-            <div className='form-builder-field' key={field.id}>
+            <div ref={this.attachFieldRef} className='form-builder-field' key={field.id}>
```

Comment:
I don't understand how this can work. All fields will trigger attachFieldRef even when rendering. How to make sure this will only affect the newly added field?